src/config/index.ts

```diff
@@ -31,6 +29,15 @@ export const REFERENCE = "OverseasEntitiesReference";
 export const PAYMENT_REQUIRED_HEADER = "x-payment-required";
 export const PAYMENT_PAID = "paid";
 
+//  FEATURE FLAGS
+const envFlagSfx = 'FEATURE_FLAG_';
+enum FeatureFlag {
+  ENABLE_SHOW_SERVICE_OFFLINE_PAGE = <any>getEnvironmentValue("SHOW_SERVICE_OFFLINE_PAGE"),
+  ENABLE_REFRESH_TOKEN_29092022 = <any>getEnvironmentValue(`${envFlagSfx}ENABLE_REFRESH_TOKEN_29092022`, "false"),
```

Comment:
If remaining (and whilst we're refactoring this), what do we think about removing the date at the end of the Node JS constant names? e.g. say:

`ENABLE_REFRESH_TOKEN = <any>getEnvironmentValue(`${envFlagSfx}ENABLE_REFRESH_TOKEN_29092022`, "false"),`

The date is needed at the end of the feature flag name in config files (according to our standards) but I don't see the need for its presence and the increased verbosity when used throughout the code.

Reply:
Agree fixed

Could you please explain why need DATE at the end of feature flag env variable?

Reply:
It is the companies house standard - https://github.com/companieshouse/styleguides/blob/main/feature_flag.md

Reply:
Thank you, but this do not explain why need date
We have git blame to see when evn variable appear in project if will be need this information.

Reply:
Not sure why the date is part of the standard but can see it could be useful, e.g. having date in the name makes it immediately obvious how old a feature (flag) is, especially when viewing that feature flag in a config flag in a remote environment file that doesn't have any Git history behind it.